src/main/java/com/deusgmbh/xcusatio/api/services/RNVAPI.java

```diff
@@ -9,7 +15,42 @@
     @Override
     public RNVContext get(UserSettings usersettings) {
         // TODO Auto-generated method stub

```

Comment:
Kann weg oder?

Reply:
Ach ne unten ist nur eine Beispielliste, dann schreib in das TODO vlt rein, dass noch die richtige API Abfrage kommen soll